### DISCUSSION TITLE
Zepp OS 2.1 New Features

### DISCUSSION THREAD

**Discussion by MyoungXUE** [zepp-health/zeppos-forum#151] [Announcements]
The Amazfit T-Rex Ultra is powered by Zepp OS 2.1

In Zepp OS 2.1, we support a number of features, including Geolocation and Barometer functionality

See https://docs.zepp.com/docs/guides/version-info/new-features-21/ for details of the update

**Comment by MyoungXUE** [comment]
More models will support Zepp OS 2.1, so stay tuned!

**Reply by Fealzhang** [reply]
Are there plans to upgrade GTR3?

**Reply by alexmathew11** [reply]
Yeah might be

**Reply by MyoungXUE** [reply]
> Are there plans to upgrade GTR3?

Hello, due to the performance limitations of GT3 series hardware, it cannot upgrade to Zepp OS 2.x system.

**Comment by Fealzhang** [comment]
Not even the gtr3 pro? That's a great pity.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Myoung XUE - Zepp ***@***.***&gt;; 
发送时间: 2023年5月4日(星期四) 中午11:56
收件人: ***@***.***&gt;; 
抄送: "Marry ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [zepp-health/zeppos-forum] Zepp OS 2.1 New Features (Discussion #151)





  
Are there plans to upgrade GTR3?
  
Hello, due to the performance limitations of GT3 series hardware, it cannot upgrade to Zepp OS 2.x system.

**Comment by joaquinperaza** [comment]
Will GTR4 support API 2.1?

**Reply by alexmathew11** [reply]
Yes of course

**Comment by ThinkAndFly** [comment]
Hi! Is the GTR2 (2022 Version) Discontinued?  In developer mode I see it has API 1.0 but the software version says 2.2.2.5.

Will it get replying capabilities such as reply to a message or make a call?
